PMMain/Midas/MidasGUI/GestureFilter.cpp

```diff
@@ -261,7 +261,7 @@ void GestureFilter::registerStateSequences(void)
     // From Keyboard:
     toLockResponse.name = "Keyboard To Lock"; // OVERRIDE for Keyboard.
     toLockSeq[0] = (SeqElement(Pose::Type::thumbToPinky));
-    toLockSeq[1] = (SeqElement(Pose::Type::waveOut));
+    toLockSeq[1] = (SeqElement(Pose::Type::waveIn));
```

Comment:
as described in person, can't have a wave-in start since that's for backspace currently. could change later.